server/services/matchChat.ts

```diff
@@ -0,0 +1,58 @@
+import { IMatchMessage } from '../interfaces';
+import { Message } from '../models';
+import { CustomError } from '../utils';
+
+const addMessageService = async ({
```

Comment:
add the returning types for all services on this file: Promise<{
"your types for the response object"
}>